### PR TITLE
fix issue #1029

### PR DESCRIPTION
**Pull request by suppermancool**
fix issue #1029
Chrome-Should not allow to add same website in the “Linked Account”, By one with Lower case and Another one in upper case

### CODE REVIEW

File: src/shared/components/Settings/Account/LinkedAccount/AddWebLink.jsx [hunk 1]
```diff
@@ -92,6 +103,16 @@ Please enter a valid URL
                   </div>
                 )
               }
+              {
+                isWebLinkExist
+                && (
+                  <div styleName="form-input-error">
+                    <p>
+                      {`You've already added link ${webLink}`}
```
[the remark this answers is not in the document]
Comment (suppermancool) [reply src/shared/components/Settings/Account/LinkedAccount/AddWebLink.jsx]: @ThomasKranitsas done

File: src/shared/components/Settings/Account/LinkedAccount/AddWebLink.jsx [hunk 2]
```diff
@@ -54,10 +56,19 @@ export default class AddWebLink extends React.Component {
     return !webLink || /^(http(s?):\/\/)?(www\.)?[a-zA-Z0-9\.\-\_]+(\.[a-zA-Z]{2,15})+(\/[a-zA-Z0-9\_\-\s\.\/\?\%\#\&\=]*)?$/.test(webLink); /* eslint-disable-line no-useless-escape */
   }
 
+  isWebLinkExist() {
```
[the remark this answers is not in the document]
Comment (suppermancool) [reply src/shared/components/Settings/Account/LinkedAccount/AddWebLink.jsx]: @ThomasKranitsas done

[the remark this answers is not in the document]
Comment (ThomasKranitsas) [reply src/shared/components/Settings/Account/LinkedAccount/AddWebLink.jsx]: I still see the `isWebLinkExist`. Did you push your changes?

[the remark this answers is not in the document]
Comment (ThomasKranitsas) [reply src/shared/components/Settings/Account/LinkedAccount/AddWebLink.jsx]: sorry, I see it's the 3rd commit

**Pull request merged**